api/c/indigo/src/indigo_molecule.cpp

```diff
@@ -36,6 +36,7 @@
 #include "molecule/query_molecule.h"
 #include "molecule/smiles_loader.h"
 #include "molecule/smiles_saver.h"
+#include <unordered_set>
```

Comment:
This probably should be deleted (see the comment below), but in general we need to follow a special includes order:

1. special header for current cpp file (if any)

2. standard library headers in alphabetic order

3. other third-party headers in alphabetic order

4. indigo-core headers in alphabetic order

5. current project headers in alphabetic order.

Reply:
Hmmm, I always see that standard headers (or outside headers in <>) are placed to the end, since (if they are not indirectly included in headers above)  they are very implementation-specific. 
Anyway, they will be reordered according to rules, right?

Reply:
It is being done automatically only by clang-tidy, in case it's correctly set up. We don't use it yet, so for now we should do it manually. clang-format knows nothing about standard library, first-party or third-party code, and only sorts headers that are not splitted with empty line in alphabetical order.